### ISSUE TITLE
Using quickstart instructions can result in errors (when it's not the first time)

### ISSUE THREAD

**Issue by hugobowne** [coiled/coiled-issues#57]
I decided to follow the quickstart guide again today to get a sense for user flow.

```
coiled login --token XXXXXXX
```
returned
```
Usage: coiled login [OPTIONS]
Try "coiled login -h" for help.
Error: no such option: --token
```
then after `pip install coiled --upgrade` (which @mrocklin suggested I do; I didn't immediately know to include `--upgrade`),
I got 
```
Traceback (most recent call last):
  File "/Users/hugobowne-anderson/opt/anaconda3/bin/coiled", line 8, in <module>
    sys.exit(cli())
  File "/Users/hugobowne-anderson/opt/anaconda3/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/Users/hugobowne-anderson/opt/anaconda3/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Users/hugobowne-anderson/opt/anaconda3/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/hugobowne-anderson/opt/anaconda3/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/hugobowne-anderson/opt/anaconda3/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Users/hugobowne-anderson/opt/anaconda3/lib/python3.7/site-packages/coiled/cli/login.py", line 17, in login
    handle_credentials(server=server, token=token, save=True)
  File "/Users/hugobowne-anderson/opt/anaconda3/lib/python3.7/asyncio/base_events.py", line 583, in run_until_complete
    return future.result()
  File "/Users/hugobowne-anderson/opt/anaconda3/lib/python3.7/site-packages/coiled/utils.py", line 113, in handle_credentials
    raise Exception(data)
Exception: {'detail': 'Authentication credentials were not provided.'}
```

Matt then suggested ` rm ~/.config/dask/coiled.yaml`

After this it was smooth sailing!

**Comment by dantheman39** [comment]
I believe this is the same as [this issue](https://github.com/coiled/coiled-issues/issues/45)

**Comment by jrbourbeau** [comment]
That's correct. We just released `coiled` 0.0.23 which contains a fix for this. Thanks for reporting @hugobowne!

**Comment by hugobowne** [comment]
For sure @jrbourbeau!
